bin/test-suite/src/suite/consensus/invalid_transactions/test_invalid_pegin.rs

```diff
@@ -38,6 +38,24 @@ async fn generate_invalid_pegin_metas(
 ) -> Vec<(Vec<PeginMeta>, &'static str)> {
     let mut invalid_pegin_meta_cases = Vec::new();
 
+    // Create invalid pegin meta with invalid reference hash for v1
+    let invalid_ref_hash_meta = vec![PeginMeta::V1(PeginMetaV1 {
+        inner: PeginMetaV0 {
+            version: PEGIN_META_VERSION_V1,
+            outpoint: bitcoin::OutPoint::new(pegin_tx.compute_txid(), vout),
+            address: eth_account.clone(),
+            aggregate_publickey: secp256k1::PublicKey::from_str(
+                gateway_address_response.aggregate_public_key.as_str(),
+            )
+            .expect("valid public key"),
+            tx: pegin_tx.clone(),
+            merkle_proof: pmt.clone(),
+            block_headers: headers.clone(),
+        },
+        ref_block_hash: B256::from_slice(&[0; 32]),
+    })];
+    invalid_pegin_meta_cases.push((invalid_ref_hash_meta, "Invalid reference hash for v1"));
```

Comment:
Nit: refrence hash implies v1. 

```suggestion
    invalid_pegin_meta_cases.push((invalid_ref_hash_meta, "Invalid reference hash"));
```

Reply:
Done!